### DISCUSSION TITLE
🆕 Quick Access to Projects on GitHub Mobile 📱

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#59625] [Mobile]
<img width="694" alt="Screenshot 2023-06-29 at 12 14 44 PM" src="https://github.com/community/community/assets/95834130/9a7151a4-75dd-4d54-8fda-f0f01be95953">

Projects on GitHub Mobile are now one tap away. ✨ We've streamlined the workflow for viewing your projects with a new entry point on the Home screen to keep you updated on the go. 🏃

Read more about this exciting update in the [changelog](https://github.blog/changelog/2023-06-27-quick-access-to-projects-on-github-mobile/). 🚀 

- [Explore GitHub Mobile](https://github.com/mobile) 
- [Github Mobile Docs](https://docs.github.com/en/get-started/using-github/github-mobile)

**Comment by Akash1134** [comment]
🚀

**Comment by AmirrezaNasrollahzade** [comment]
it is cool.

**Comment by Ashegham** [comment]
حساب گوگلم و گوگل کروم کار نمکنند، انصال اینترنتم هی قطع میشه
